torchgeo/trainers/tasks.py

```diff
@@ -257,3 +261,118 @@ def configure_optimizers(self) -> Dict[str, Any]:
                 "monitor": "val_loss",
             },
         }
+
+
+class MultiLabelClassificationTask(ClassificationTask):
+    """Abstract base class for multi label image classification LightningModules."""
```

Comment:
This is not an ABC (doesn't inherit from `abc.ABC`) and I don't think it needs to be. Why can't `num_classes` be an arg passed to `__init__`?

Reply:
It definitely can be. Will refactor to do so.

Reply:
There's not a great way to do this w/o adding more boilerplate everywhere. So I'll add this in a future PR.

Reply:
Can you explain this in more detail? Why can't it just be an arg to `__init__`?